draft-ietf-rtcweb-jsep.xml

```diff
@@ -3901,63 +3901,59 @@ v=0
 o=- 4962303333179871722 1 IN IP4 0.0.0.0
 s=-
 t=0 0
-a=group:BUNDLE a1 v1
 a=ice-options:trickle
+a=group:BUNDLE a1 v1
 m=audio 56500 UDP/TLS/RTP/SAVPF 96 0 8 97 98
 c=IN IP4 192.0.2.1
 a=mid:a1
-a=rtcp:56501 IN IP4 192.0.2.1
 a=msid:47017fee-b6c1-4162-929c-a25110252400
-       f83006c5-a0ff-4e0a-9ed9-d3e6747be7d9
 a=sendrecv
 a=rtpmap:96 opus/48000/2
 a=rtpmap:0 PCMU/8000
 a=rtpmap:8 PCMA/8000
 a=rtpmap:97 telephone-event/8000
 a=rtpmap:98 telephone-event/48000
 a=maxptime:120
-a=ice-ufrag:ETEn1v9DoTMB9J4r
+a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
+a=extmap:2 urn:ietf:params:rtp-hdrext:sdes:mid
+a=ice-ufrag:ETEn
 a=ice-pwd:OtSK0WpNtpUjkY4+86js7ZQl
 a=fingerprint:sha-256
-              19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04
-             :BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
+              19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04:
+              BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
 a=setup:actpass
+a=dtls-id:1
+a=rtcp:56501 IN IP4 192.0.2.1
 a=rtcp-mux
 a=rtcp-rsize
-a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
-a=extmap:2 urn:ietf:params:rtp-hdrext:sdes:mid
-a=candidate:3348148302 1 udp 2113937151 192.0.2.1 56500
-            typ host
-a=candidate:3348148302 2 udp 2113937151 192.0.2.1 56501
-            typ host
+a=candidate:1 1 udp 2113929471 192.0.2.1 56500 typ host
+a=candidate:1 2 udp 2113929470 192.0.2.1 56501 typ host
 a=end-of-candidates
 
 m=video 56502 UDP/TLS/RTP/SAVPF 100 101
 c=IN IP4 192.0.2.1
-a=rtcp:56503 IN IP4 192.0.2.1
 a=mid:v1
 a=msid:61317484-2ed4-49d7-9eb7-1414322a7aae
-       f30bdb4a-5db8-49b5-bcdc-e0c9a23172e0
 a=sendrecv
 a=rtpmap:100 VP8/90000
 a=rtpmap:101 rtx/90000
 a=fmtp:101 apt=100
-a=ice-ufrag:BGKkWnG5GmiUpdIV
+a=extmap:3 urn:ietf:params:rtp-hdrext:sdes:mid
```

Comment:
I thought there was some reason they needed to be unique. But TBH I am not 100% sure.

What does the attribute grouping doc say?